sys/net/gnrc/network_layer/icmpv6/error/gnrc_icmpv6_error.c

```diff
@@ -27,7 +27,9 @@
 #define ICMPV6_ERROR_SET_VALUE(data, value) \
     ((icmpv6_error_pkt_too_big_t *)(data))->mtu = byteorder_htonl(value)
 
+#ifndef MIN
```

Comment:
Maybe rather use
```suggestion
#undef MIN  /* undefine common name just to be safe */
```
in case `MIN` is not actually determining the minimum between two values (could also be just `60` for MINutes?

Reply:
Good point.